fetch_flow_definitions/src/firestore_wrapper.py

```diff
@@ -0,0 +1,34 @@
+import firebase_admin
```

Comment:
Does this belong here? I can't immediately think of a better place to put it, but it feels like if we're starting to share firebase wrapper code between multiple projects it might make sense to factor it out into a shared library?

Reply:
I was wondering the same. Pipeline-Infrastructure maybe?

Reply:
(In any, case requires future PRs)